### PR TITLE
Refine flowgraph nodes for readability

### PR DESCRIPTION
**Pull request by XuanYang-cn**
Signed-off-by: yangxuan <xuan.yang@zilliz.com>

/kind improvement

### CODE REVIEW

File: internal/datanode/flow_graph_insert_buffer_node.go [hunk 1]
```diff
@@ -246,6 +167,50 @@ func (ibNode *insertBufferNode) Operate(in []Msg) []Msg {
...
+
+	for _, sp := range spans {
+		sp.Finish()
+	}
+
+	// send delete msg to DeleteNode
+	return []Msg{&res}
+}
+
+func (ibNode *insertBufferNode) varifyInMsg(in []Msg) (*flowGraphMsg, bool) {
```
Comment (congqixia) [inline internal/datanode/flow_graph_insert_buffer_node.go:193]: NIT: maybe verify?

**Comment by XuanYang-cn** [comment]
rerun ut

**Comment by XuanYang-cn** [comment]
rerun ut

**Comment by sunby** [comment]
fix https://github.com/milvus-io/milvus/issues/19899

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**